### PR TITLE
CI Tests for component-updates (test)

### PR DESCRIPTION
**Pull request by heoelri**
@msimecek / @nielsams

### CODE REVIEW

**Comment by heoelri** [comment]
Testing #186

**Comment by heoelri** [comment]
E2E was successful and CI build tests, too. @nielsams any additional tests that need to be done before we can merge this?

#189 / #188 / #180 

https://dev.azure.com/Mission-Critical/Online/_build/results?buildId=503&view=results

**Comment by heoelri** [comment]
> E2E was successful and CI build tests, too. @nielsams Niels Buit FTE any additional tests that need to be done before we can merge this?
> 
> #189 / #188 / #180
> 
> https://dev.azure.com/Mission-Critical/Online/_build/results?buildId=503&view=results

https://github.com/Azure/Mission-Critical-Online/runs/5445951397?check_suite_focus=true

**Comment by nielsams** [comment]
Last time the issue was having front door in front of grafana, with the different host name. If you can log in to grafana after deployment it should be fine.

**Comment by heoelri** [comment]
Thanks @nielsams and sorry for bothering you during your vacation!

I noticed a "origin not allowed" error after logging into grafana. Is this what you're refering to?

File: src/infra/monitoring/grafana/Dockerfile [hunk 1]
```diff
@@ -18,7 +18,7 @@ RUN chmod +x insertqueries.sh && ./insertqueries.sh
 # Build phase is done, now build the container. Start with the office Grafana image
 # We're using 8.3.4 for now since in later versions our frontdoor origin is not accepted. 
 # This is a known issue and we'll upgrade to a later version once that's resolved. 
-FROM grafana/grafana:8.3.4
+FROM grafana/grafana:8.4.2
```
Comment (heoelri) [inline src/infra/monitoring/grafana/Dockerfile:21]:
```suggestion
FROM grafana/grafana:8.3.4
```

Comment (heoelri) [inline src/infra/monitoring/grafana/Dockerfile]:
```suggestion
FROM grafana/grafana:8.4.2
```

**Comment by nielsams** [comment]
> Thanks @nielsams and sorry for bothering you during your vacation!
> 
> 
> 
> I noticed a "origin not allowed" error after logging into grafana. Is this what you're refering to?

Yes that's the one. If we stay on current grafana version, that's not an issue.

**Comment by heoelri** [comment]
https://github.com/grafana/grafana/issues/46321

**Comment by sebader** [comment]
@heoelri is this good to merge now?

**Comment by heoelri** [comment]
@sebader I'll close this PR for now. Still hoping to get a response for our issue in the grafana repo.

**Pull request closed without merging**